src/Resources/config/services.yml

```diff
@@ -0,0 +1,50 @@
+services:
+    contao.backend.subscriber:
```

Comment:
I agree on that one.
Using the prefix `cca` might be good for all extensions by us.
Suffixing it by the bundle name is also a neat idea. :+1: on that one.